File: test/unit/method_test.rb
#!/usr/bin/env ruby

#$use_tree=true
require_relative '../parser_test_helper'

class MethodTest < ParserBaseTest

  include ParserTestHelper

  def test_result
    parse "show 3"
    assert_equals result,"3"
    parse "how to test:show 3;ok"
    assert methods.count>0
    assert_equals methods["test"].body,"show 3;"
    parse "test"
    assert_equals result,"3"
    #assert @variables['x']==3
  end

end

File: angelos/test/method_test.py
import _global
from parser_test_helper import *


class MethodTest(ParserBaseTest):
    

    def test_result(self):
        parse('alias show puts;show 3')
        assert_equals(result(), '3')
        parse('how to test:show 3;ok')
        assert(bigger_than(0, len(methods(), )))
        assert_equals(body(methods['test'], ), 'show 3;')
        parse('test')
        assert_equals(result(), '3')